generators/plugin/index.js

```diff
@@ -98,7 +105,7 @@ module.exports = yeoman.Base.extend({
 
     this.registerTransformStream(filters);
 
-    this.fs.copyTpl([`${this.templatePath()}/**/*`, adminFiles, publicFiles], this.destinationPath(), _props);
+    this.fs.copyTpl([`${this.templatePath()}/**/*`, adminFiles, publicFiles, composerFile], this.destinationPath(), _props);
```

Comment:
Yes it matches, but it also matches for admin and public files. The reason behind this is to exclude on demand.